packages/library-base/src/commonMain/kotlin/io/realm/kotlin/dynamic/DynamicRealmObject.kt

```diff
@@ -197,6 +198,16 @@ public interface DynamicRealmObject : BaseRealmObject {
      * property's [RealmStorageType.kClass].
      */
     public fun getObjectSet(propertyName: String): RealmSet<out DynamicRealmObject>
+
+    /**
+     * Returns a linking objects collection referenced by the property name as a [RealmResults].
+     *
+     * @param propertyName the name of the linking objects property to retrieve for.
+     * @return the referencing objects as a [RealmResults].
+     * @throws IllegalArgumentException if the class doesn't contain a field with the specific
+     * name, if trying to retrieve values for non linking objects properties.
+     */
+    public fun getLinkingObjects(propertyName: String): RealmResults<out DynamicRealmObject>
```

Comment:
I believe so, this property name is not the target property name but the `linkingObjects` property name. It is equivalent to using the source class and source property. 

Because the linking objects property is defined in the schema we can use it directly here.